lnbits/extensions/tipjar/views_api.py

```diff
@@ -55,25 +54,32 @@ async def api_create_tip(data: createTips):
             status_code=HTTPStatus.NOT_FOUND, detail="Tipjar does not exist."
         )
 
-    webhook = tipjar.webhook
-    charge_details = await get_charge_details(tipjar.id)
+    wallet_id = tipjar.wallet
+    wallet = await get_wallet(wallet_id)
+    if not wallet:
+        raise HTTPException(
+            status_code=HTTPStatus.NOT_FOUND, detail="Tipjar wallet does not exist."
+        )
+
     name = data.name
+
     # Ensure that description string can be split reliably
     name = name.replace('"', "''")
     if not name:
         name = "Anonymous"
+
     description = f"{name}: {message}"
     charge = await create_charge(
-        user=charge_details["user"],
+        user=wallet.user,
```

Comment:
a bit hard to follow, but tests passed

Reply:
i removed the helper, which was bloated and only was used in 1 place, that simplified the whole part of the code a lot.